src/lib/jws/jwsValidator.js

```diff
@@ -39,9 +39,9 @@ class JwsValidator {
      */
     validate(request) {
         try {
-            const { headers, body } = request;
+            const { headers, body, data } = request;
```

Comment:
Consider doing this:
```suggestion
            const { headers, body, data } = request;
            const payload = body || data;
```
and using `payload` subsequently.

Reply:
done